src/testsuite/pveclib_test.c

```diff
@@ -22,7 +22,7 @@
 
 #define __STDC_WANT_DEC_FP__    1
 
-#ifndef __clang__
+#ifndef PVECLIB_DISABLE_F128MATH
 /* Disable <stdlib.h> for __clang__ because of bug involving <floatn.h>
    incombination with -mcpu=power9 -mfloat128 */
```

Comment:
delete this comment?

Reply:
I think I will keep this.

It is not obvious how the problem with math.h is related to stdlib.h